include/for_each.hpp

```diff
@@ -0,0 +1,113 @@
+#pragma once
+
+#include <utility>
+
+#include "ddc/chunk_span.hpp"
+#include "ddc/discrete_coordinate.hpp"
+#include "ddc/discrete_domain.hpp"
+
+/// Serial execution
+struct serial_policy
```

Comment:
you could create a constexpr instance for the users to use

Reply:
Done